tests/test_frontdoor.py

```diff
@@ -1526,6 +1526,46 @@ def test_on_check_answers_page_changing_to_no_loft_hides_loft_follow_up_question
     _assert_change_button_is_hidden(page, "loft-insulation")
 
 
+@pytest.mark.parametrize(
+    ("contact_number", "expect_to_accept"),
+    [
+        ("01632 960 001", True),
+        ("07700 900 982", True),
+        ("+44 808 157 0192", True),
+        ("+33 06 12 34 56 78", True),
+        ("1", False),
+        ("not a number", False),
```

Comment:
I'd really like this to have quite thorough tests. There's a bunch of valid UK numbers [here](https://gist.github.com/edwardhorsford/a9df6b16a561cd54336fbba51572db25) (linked from one of the GDS discussions) - could we cover all of those?

Reply:
sure